src/components/anchor/code/views/anchorView.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+var DataAttributesMixin = require('react-data-attributes-mixin');
+
+module.exports = React.createClass({
+
+  mixins: [DataAttributesMixin],
+
+  getDefaultProps: function() {
+    return {
+      href: '#'
+    };
+  },
+
+  propTypes: {
+    children: React.PropTypes.node,
+    data: React.PropTypes.object,
+    href: React.PropTypes.string,
+    target: React.PropTypes.oneOf(['self', 'blank', 'parent', 'top']),
```

Comment:
Should this be required? I wouldn't think we'd always want to pass in a target seeing as it's not a required attribute on the W3 spec for anchor?